packages/@aws-cdk/aws-ecs/lib/ec2/ec2-service.ts

```diff
@@ -196,18 +196,31 @@ export class Ec2Service extends BaseService implements IEc2Service, elb.ILoadBal
    * Don't call this. Call `loadBalancer.addTarget()` instead.
    */
   public attachToClassicLB(loadBalancer: elb.LoadBalancer): void {
+    return this.defaultCLBLoadBalancerTarget.attachToClassicLB(loadBalancer);
+  }
+
+  /**
+   * This method is called to register the specified ECS target in this service to a target group.
+   *
+   * Don't call this function alone. Instead, call `lb.addTarget(service.loadBalancerTarget())`
+   * to add this target to a load balancer.
+   */
+  public classicLoadBalancerTarget(options: LoadBalancerTargetOptions): elb.ILoadBalancerTarget {
```

Comment:
As mentioned before, I think we should be able to combine this with the other `loadBalancerTarget()`.